addons/iot_box_image/configuration/checkout.sh

```diff
@@ -14,6 +14,17 @@ git fetch "${localremote}" "${localbranch}" --depth=1
 git reset "${localremote}"/"${localbranch}" --hard
 sudo git clean -dfx
 
+# Update requirements
+pip3 install -r /home/pi/odoo/addons/iot_box_image/configuration/requirements.txt --user --break-system-package
+
+# Update packages: install new ones, upgrade existing ones in the background
+sudo chroot /root_bypass_ramdisks/ /bin/bash <<'EOF'
+mount -t proc proc /proc
+apt-get udpate
+xargs apt-get -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" install < /home/pi/odoo/addons/iot_box_image/configuration/packages.txt
+nohup bash -c "apt-get upgrade -y && apt-get -y autoremove" > /dev/null 2>&1 &  # Run in background to avoid blocking the script
+EOF
```

Comment:
same here, let's check if the file exists to avoid:

```
E: Invalid operation udpate
/bin/bash: line 3: /home/pi/odoo/addons/iot_box_image/configuration/packages.txt: No such file or directory
```